lib/note-content-editor.tsx

```diff
@@ -132,6 +133,11 @@ class NoteContentEditor extends Component<Props> {
         }
       }, SPEED_DELAY);
     }
+    if (prevProps.lineLength !== this.props.lineLength) {
+      setTimeout(() => {
+        window?.editor.layout();
+      }, 400);
+    }
```

Comment:
why the timeout?

Reply:
Doesn't work without the timeout. I think it happens too soon for some reason. Not sure what is going on there but I figured I'd get what I had working up before going deeper.

Reply:
did you try `window.editor?.updateOptions({ lineHeight: this.props.lineLength })`

Reply:
That does not work either without the timeout.

Reply:
strange. I'll try and examine this too. something smells fishy about needing that timeout and I want to make sure we don't overlook something important